website/newsletters/emails.py

```diff
@@ -38,32 +33,23 @@ def send_newsletter(newsletter):
     main_partner = Partner.objects.filter(is_main_partner=True).first()
     local_partners = services.split_local_partners()
 
-    with mail.get_connection() as connection:
-        language = ("en", "English")
-        translation.activate(language[0])
-
-        context = {
-            "newsletter": newsletter,
-            "agenda_events": events,
-            "main_partner": main_partner,
-            "local_partners": local_partners,
-            "lang_code": language[0],
-        }
-        context.update(thumbnail_sizes(None))
-
-        try:
-            send_email(
-                to=[f"newsletter@{settings.GSUITE_DOMAIN}"],
-                subject=newsletter.title,
-                txt_template="newsletters/email.txt",
-                html_template="newsletters/email.html",
-                from_email=settings.NEWSLETTER_FROM_ADDRESS,
-                connection=connection,
-                context=context,
-            )
-
-            logger.info("Sent %s newsletter", language[1])
-        except SMTPException:
-            logger.exception("Failed to send the %s newsletter", language[1])
+    context = {
+        "newsletter": newsletter,
+        "agenda_events": events,
+        "main_partner": main_partner,
+        "local_partners": local_partners,
+    }
+
+    try:
+        send_email(
+            to=[f"newsletter@{settings.GSUITE_DOMAIN}"],
```

Comment:
Maybe import from mailinglist app here

Reply:
Best we could do is take the list from https://github.com/svthalia/concrexit/blob/71936f46d69b275e6dcdc450e5312daf9f23fadd/website/mailinglists/services.py#L183, but thats a dict by name, so to get the item we'd need the name which is directly the 'user' part of the address. So there's little to do